statusHandler/termuic/termuiRenders/widgetsRender.go

```diff
@@ -184,14 +191,47 @@ func (wr *WidgetsRender) prepareChainInfo() {
 	wr.chainInfo.Rows = rows
 }
 
+func (wr *WidgetsRender) prepareBlockInfo() {
+	//7 rows and one column
+	numRows := 7
+	rows := make([][]string, numRows)
+
+	currentBlockHeight := wr.getFromCacheAsUint64(core.MetricSynchronizedRound)
```

Comment:
I think this should be MetricNonce and the following 2 lines (comment + increment) should be deleted

Reply:
yes, it is not correct like it is now. It should be at least MetricCurrentRound+1

Reply:
done.